account/update_account_with_account_key_role_based/boilerplate.js

```diff
@@ -0,0 +1,111 @@
+const path = require('path')
+const dotenv = require('dotenv')
+const Caver = require('caver-js')
+const ROOT_DIR = path.join(__dirname, '../..'); // Path can be changed based on its actual location.
```

Comment:
```suggestion
const ROOT_DIR = path.join(__dirname, '../..') // Path can be changed based on its actual location.
```
Can you change this to the other files too?; sorry for bothering you